modules/rostests/apitests/shell32/ShellHook.cpp

```diff
@@ -357,65 +305,43 @@ WindowProc(HWND hwnd, UINT uMsg, WPARAM wParam, LPARAM lParam)
 {
     if (uMsg == s_uShellHookMsg && uMsg != 0)
     {
-        DWORD style, exstyle, owner_style, owner_exstyle;
-        HWND hwndOwner;
-        DWORD dwFlags;
         switch (wParam)
         {
             case HSHELL_WINDOWCREATED:
                 if ((HWND)lParam != s_hwndTarget)
                     break;
-                style = (LONG)GetWindowLongPtrW(s_hwndTarget, GWL_STYLE);
-                exstyle = (LONG)GetWindowLongPtrW(s_hwndTarget, GWL_EXSTYLE);
-                if (style & WS_CHILD)
-                    hwndOwner = GetParent(s_hwndTarget);
-                else
-                    hwndOwner = GetWindow(s_hwndTarget, GW_OWNER);
-                owner_style = (LONG)GetWindowLongPtrW(hwndOwner, GWL_STYLE);
-                owner_exstyle = (LONG)GetWindowLongPtrW(hwndOwner, GWL_EXSTYLE);
-                dwFlags = (1 << 0);
```

Comment:
So if I understand correctly, using window styles for comparisons was not a reliable test?

Reply:
This was excessive tests.